src/analysis/normed_space/finite_dimension.lean

```diff
@@ -382,3 +380,28 @@ instance finite_dimensional.proper_real
 finite_dimensional.proper ℝ E
 
 attribute [instance, priority 900] finite_dimensional.proper_real
+
+/-- Due to the way we define `summable`, in a finite dimensional normed space over `ℝ`, a series
+`∑ x, f x` is summable if and only if `∑ x, ∥f x∥` is summable. -/
```

Comment:
I don't understand this "Due to the way we define". It sounds like mathlib has a non-standard definition here. You should probably also point out in this docstring that one implication is a characterization of completeness and the other one is a characterization of finite dimensionality, even if we may not have proven this.

Reply:
For a 1st year undergraduate student, `∑ k, f k` converges iff the sequence `∑_{k=1}^{n} f k` has a limit. We use a much more general definition. I'll try to write a better docstring.

Reply:
In this particular case there is no risk of confusion since the source of `f` is not ordered. And again the definition of summable is *not* more general in the case of function with source `nat`, it's more restrictive. At some point we'll have an API for the elementary notion.

Reply:
Yes, I meant more general as in "works for any domain". How would you call `def`s for the elementary version?

Reply:
Updated docstring.